### PR TITLE
docs(dotnet): First part/pass for guides

### PR DESCRIPTION
**Pull request by avodovnik**

### CODE REVIEW

File: docs/src/auth.md [hunk 1]
```diff
@@ -71,6 +71,18 @@ page.click('text=Submit')
 # Verify app is logged in
 ```
 
+```csharp
+var page = await browser.NewPageAsync();
+await page.GoToAsync("https://github.com/login");
```
Comment (pavelfeldman) [inline docs/src/auth.md:76]: GoToAsync is now Goto

File: docs/src/auth.md [hunk 2]
```diff
@@ -290,6 +329,20 @@ with sync_playwright() as p:
...
+```csharp
+using Microsoft.Playwright;
+
+class Guides
+{
+  public async void Main()
+  {
+      using var playwright = await Playwright.CreateAsync();
+      var chromium = playwright.Chromium;
+      var context = chromium.LaunchPersistentContextAsync(@"C:\path\to\director\", headless: false);
```
Comment (pavelfeldman) [inline docs/src/auth.md:341]: director -> directory

File: docs/src/browsers.md [hunk 3]
```diff
@@ -72,6 +72,19 @@ Using `firefox-stable` is a 2-steps process:
     browser = playwright.firefox.launch(channel="firefox-stable")
     ```
 
+    ```csharp
```
Comment (pavelfeldman) [inline docs/src/browsers.md:75]: I'll nuke this section

File: docs/src/auth.md (hunk 1 above)
Comment (avodovnik) [reply docs/src/auth.md:76]: Yeah, I was going to go and retrospectively fix them all in one swoop down the line. The fact it's still here is a result of me checking/compiling the samples as I write them, and I have an _old_-ish version of the API.

**Pull request merged**